NEWS.md

```diff
@@ -1,3 +1,7 @@
+## 29.3.2 2020-03-18
+
```

Comment:
Normally we wait with updating the NEWS file. More issues or date might be incorrect.